documentation/reference/query-language-reference.html

```diff
@@ -864,6 +923,10 @@ <h2 id="where">where</h2>
       <li> The standard <em>label</em> annotation may be used to mark the query operator with a label that can be
            referred to from the ranking expression in the rank profile. See the documentation for the
            <a href="rank-features.html#closeness(dimension,name)">closeness</a> rank feature.</li>
+      <li> The <em>distanceThreshold</em> annotation may be used to filter away hits with a higher distance
+           than the given threshold from the results.  Note that you will never get more hits with
+           <em>distanceThreshold</em> than you would get without it; if you want more hits you need to
+           increase the <em>targetHits</em> value also.
```

Comment:
Perhaps we should add a note that this threshold must correspond to the range of values output by the distance metric specified (with link to "schema-reference.html#distance-metric")?